File: C#/Mit Polymorphismus/Hund.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

class Hund : Haustier
{
    public Hund(string n) : base(n)
    {}
    public override void Fressen()
    {
        System.Console.WriteLine("Knochen nagen");
    }

    public override void Freuen()
    {
        System.Console.WriteLine("wedeln");
    }

    public override void LautGeben()
    {
        System.Console.WriteLine("bellen");
    }
}

File: Java/Mit Polymorphismus/Hund.java
public class Hund extends Haustier{
    public Hund(){
        name = "";
    }
    public Hund(String n){
        name = n;
    }

    public void fressen(){
        System.out.println("Knochen nagen");
    }

    public void freuen(){
        System.out.println("wedeln");
    }

    public void lautGeben(){
        System.out.println("bellen");
    }

}
